### PR TITLE
Title enhancement, OSVDB refs

### PR DESCRIPTION
**Pull request by dfreamon**

### CODE REVIEW

**Comment by bcook-r7** [comment]
Thanks @dfreamon

**Comment by wvu-r7** [comment]
@dfreamon: Please write a description next time!

File: modules/exploits/windows/antivirus/symantec_endpoint_manager_rce.rb [hunk 1]
```diff
@@ -35,6 +35,8 @@ def initialize(info = {})
           [ 'CVE', '2013-5014' ],
           [ 'CVE', '2013-5015' ],
           [ 'EDB', '31853'],
+          [ 'OSVDB', '103305'],
```
Comment (wvu-r7) [inline modules/exploits/windows/antivirus/symantec_endpoint_manager_rce.rb]: Mismatched whitespace in ```References``` is unfortunately pervasive.

Comment (wvu-r7) [inline modules/exploits/windows/antivirus/symantec_endpoint_manager_rce.rb]: Fixed. Was bugging me.

**Comment by wvu-r7** [comment]
https://github.com/rapid7/metasploit-framework/commit/a4531e62a0272b80d33002b2aef2374f70280add

File: modules/exploits/windows/antivirus/symantec_endpoint_manager_rce.rb (hunk 1 above)
Comment (bcook-r7) [inline modules/exploits/windows/antivirus/symantec_endpoint_manager_rce.rb]: Argh, sorry I missed that.

Comment (wvu-r7) [inline modules/exploits/windows/antivirus/symantec_endpoint_manager_rce.rb]: Nah, I'm just picky. :)

**Pull request merged**